.gitignore

```diff
@@ -141,6 +141,7 @@ test/dask-worker-space/
 /dask-worker-space/
 .vscode/
 generated/
+.ipynb_checkpoints/
```

Comment:
Normally an `ipynb_checkpoints` folder shouldn't be auto-generated in the root dir, since we don't have any notebook files there. Probably no need for this.

Reply:
This syntax (without leading `/`) is to ignore any checkpoints folder, not just at root.